autocomplete.ts

```diff
@@ -96,7 +97,7 @@ export default function autocomplete<T extends AutocompleteItem>(settings: Autoc
 
     const input: HTMLInputElement = settings.input;
 
-    container.className = "autocomplete " + (settings.className || "");
+    container.className = settings.className ? settings.className : "autocomplete ";
```

Comment:
should you append a space to the className?

Reply:
Nope, I missed that. Just updated.

Reply:
just to make sure: i meant the opposite (that we might need the space in both cases in the case of concatenations, don't we?)

Reply:
Oh I understand now. No, I don't see anywhere that the container's class name is modified after creation, the space was only there because the optional className was appended to the default one. We're using our own stylesheet and I wanted to cut down on the amount of extra classnames to avoid style bleed-in. 
However I did realize that I broke up/down scrolling by changing the selected class name, so I'm updating that as well now.